lib/containers/utils.pm

```diff
@@ -82,7 +82,7 @@ sub basic_container_tests {
     my $tumbleweed = "registry.opensuse.org/opensuse/tumbleweed";
 
     # Test search feature
-    validate_script_output("$runtime search --no-trunc tumbleweed", sub { m/Official openSUSE Tumbleweed images/ });
+    validate_script_output("$runtime search --no-trunc --format \"table {{.Name}} {{.Description}}\" tumbleweed | grep \"docker.io/opensuse/tumbleweed\"", sub { m/Official openSUSE Tumbleweed images/ });
```

Comment:
I vote for removing grep . It suppose to work without grep . If for some reason "docker.io/opensuse/tumbleweed" will be missing I think it might help to see what is actual output . with this grep we won't see it . If you afraid to catch "Official openSUSE Tumbleweed images" not from "docker.io/opensuse/tumbleweed" you may extend validation string instead

Reply:
Is the `grep` within a `validate_script_output` really necessary?

Reply:
Done.